lib/traject/pdc_describe_indexing_config.rb

```diff
@@ -76,10 +76,13 @@
 #   accumulator.concat [collection_name]
 # end
 
-# # ==================
-# # author fields
+# ==================
+# author fields
 
-# to_field 'author_tesim', extract_xpath("/item/metadata/key[text()='dc.contributor.author']/../value")
+to_field 'author_tesim' do |record, accumulator, _c|
+  author_name = record.xpath("/hash/creators/creator/value").map(&:text)
```

Comment:
How come we are changing this from `/item/metadata/key[text()='dc.contributor.author']/../value` to `/hash/creators/creator/value` ? Does DSpace provide a `/hash...` attribute?

Reply:
@hectorcorrea This PR isn’t for indexing DSpace content, it’s for indexing PDC Describe content. We’re taking the JSON from the PDC Describe resource and converting it to XML so we can index it with traject. An example of the file that’s created by that JSON --> XML transformation can be seen in spec/fixtures/pdc_describe/shakespeare.xml: https://github.com/pulibrary/pdc_discovery/blob/main/spec/fixtures/pdc_describe/shakespeare.xml

Reply:
We have to use the `resource` object because we need access to non-DataCite content like keywords, per conversation with @hhkitsune earlier this week.

Reply:
Bess just pointed that this code is for PDC Describe indexing not DSpace.